Update docs: `packit init` now sets specfile path in config file (#315)
packit/packit.dev: content/docs/CLI/init.md

Initiate a repository to start using packit. By default this command adds
`.packit.yaml` config file to the git repository in the current working
directory.  

> If spec file is present somewhere in your git repository, `init` will set
> the [specfile_path](/docs/configuration/#specfile_path) to it in `.packit.yaml`.
> Otherwise, `init` will set the `specfile_path` to \<the name of the repository\>.spec.

Reviewer: ```suggestion
If a spec file is found in the git repository, `init` will set
[specfile_path](/docs/configuration/#specfile_path) to point to it in `.packit.yaml`.
Otherwise, `specfile_path` is set to `<the name of the repository>.spec`.
```

Initiate a repository to start using packit. By default this command adds
`.packit.yaml` config file to the git repository in the current working
directory.  
If spec file is present somewhere in your git repository, `init` will set

> the path to it in `.packit.yaml`. Otherwise it uses git repo's name.

Reviewer 2: Reading it the first time:

- it's difficult to see how is the `specfile_path` from the next sentence related, so I'd put it into the first sentence.
- you use different tense ('will set' vs. 'uses') in each sentence 
- we can explain a bit more how is the repo's name actually used

```suggestion
the [specfile_path](/docs/configuration/#specfile_path) to it in `.packit.yaml`. Otherwise, it will use the name of the repository.
```

or

```suggestion
the [specfile_path](/docs/configuration/#specfile_path) to it in `.packit.yaml`. Otherwise, `init` will set the `specfile_path` to <the name of the repository>.spec.
```
